js/LazyContentPane.js

```diff
@@ -18,8 +18,9 @@ define([
 	"dojo/_base/declare",		// declare
 	"dijit/layout/ContentPane",	// parent
 	"dojo/dom-construct",
-	"dijit/a11yclick"			// template requirement
-], function(lang, declare, ContentPane, dconst){
+	"dijit/a11yclick",			// template requirement
+	"./es6module"
+], function(lang, declare, ContentPane, dconst, es6module){

```

Comment:
es6module parameter is in the wrong position.  It will have the value of the `dijit/a11yclick` module.